_data/hosting.yml

```
@@ -399,8 +399,8 @@ websites:
       - totp
     doc: https://ar.neolo.com/knowledgebase/215/Autenticacion-de-2-factores.html
 
-  - name: Netcup
-    url: https://netcup.eu/
+  - name: netcup
+    url: https://www.customercontrolpanel.de/
```

We usually list the main page, not the login page.
```suggestion
    url: https://netcup.eu/
```